guard-examples/library/src/main.rs

```
@@ -24,7 +24,10 @@ fn main() -> anyhow::Result<()> {
     let mut reader = Reader::new(ReadBuffer::Cursor(Cursor::new(Vec::from(
         payload.as_bytes(),
     ))));
-    let mut writer = Writer::new_with_err(WriteBuffer::Vec(vec![]), WriteBuffer::Vec(vec![]));
+    let mut writer = Writer::new_with_err(WriteBuffer::Vec(vec![]), WriteBuffer::Vec(vec![]))
+        .unwrap_or_else(|err| {
+            panic!("Error: {}", err);
```

we shouldn't be panicking here, we need to return an error

---

Whoops, misread the example.